staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -154,6 +154,9 @@ type VirtualMachineInstanceSpec struct {
 	// +listType=atomic
 	// +optional
 	AccessCredentials []AccessCredential `json:"accessCredentials,omitempty"`
+
+	// Flavor references the name of a VirtualMachineFlavor or VirtualMachineClusterFlavor
+	Flavor string `json:"flavor,omitempty" optional:"true"`
```

Comment:
this looks like it is purely informational. If the flavor is being applied to a VMI, and we want the VMI to carry knowledge of the name of the flavor that was applied, I think that belongs either in the vmi.Status section or perhaps even as an annotation for now.

Reply:
Ya I think an annotation makes sense.  The way I was thinking about this was if a user has a VM or VMI, they have a path to set `instance-type`.  The VM would do it with flavor, which sets a field on the VMI, and with only a VMI, a user can set the annotation field.